sigstore-java/src/main/java/dev/sigstore/KeylessVerifier.java

```diff
@@ -244,4 +200,59 @@ void checkCertificateMatchers(X509Certificate cert, List<CertificateMatcher> mat
           "Could not verify certificate identities: " + ce.getMessage());
     }
   }
+
+  void checkMessageSignature(
```

Comment:
@visiblefortesting?

Reply:
Not even. Should just be private

Reply:
gonna change this in the #855, the rebase is gonna be annoying.